### ISSUE TITLE
Auth request hangs, doesn't complete or return error

### ISSUE THREAD

**Issue by 0x62** [alkawryk/tinderjs#7]
This is the code I'm using to authorize it (note that it's actually part of the _this object, but it shouldn't make a difference). Before I run the auth function, I logged the params, and got the following:

```
access_token: XXXXXXXXXXXXXXXXXXXXXXXXXX, user_id: 7003430XXXXXXXX
```

So the values are correct, and in the FB response for `user_id`, `isvalid` is true, so from what I can tell there shouldn't be issues.

```
 _this.client.authorize(access_token, body.data.user_id, function(){
    console.log("Authorized");
});
```

The authorized message is never executed.

**Comment by backnight** [comment]
I have a Tinder App using TinderJS and it's no more working (for 1 week). I tried the example from this repository and it's not working anymore.

I think Tinder have made an update and this library need to be fixed.

**Comment by alkawryk** [comment]
Can you try manually adding the fix from https://github.com/alkawryk/tinderjs/pull/6 to see if it fixes the issue? If so, I will push out a new release with the pull request ASAP

**Comment by backnight** [comment]
#6 fixed my problem. Thanks.

**Comment by backnight** [comment]
Too many people abusing this mobile app with TinderJS :P need to fake the headers lol

**Comment by alkawryk** [comment]
I've published version 1.0.7 to npm now.
